### DISCUSSION TITLE
Is script.js an existing alias for plausible.js?

### DISCUSSION THREAD

**Discussion by sagarbehere** [plausible/analytics#1734] [Self-Hosted Support] [answered]
On my self-hosted instance, I can access e.g. https://my.plausible.instance/js/script.js and it seems to have the same content as https://my.plausible.instance/js/plausible.js (I am not proxying js/script.js to js/plausible.js. It seems to "just work" anyway)

Is this merely an undocumented "feature" ? Can I count on this to be preserved going forward? I'd like to use /js/script.js instead of js/plausible.js because the former is better for getting around block lists.

**Comment by ukutaht** [comment, marked as answer]
Yes, it's an undocumented feature and yes I think you can count on this going forward
